hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -225,7 +225,7 @@ public void stop() throws IOException {
   @Override
   public List<String> getRatisRoles() {
     return division.getGroup().getPeers().stream()
-        .map(peer -> peer.getAddress() == null ? "" : peer.getAddress())
+        .map(peer -> peer.getAddress() == null ? "" : peer.getAddress().concat(peer.getAdminAddress() == null ? ":FOLLOWER" : ":LEADER"))
```

Comment:
I am not sure if we should rely on the getAdminAddress() call to be null to determine if this node is LEADER or FOLLOWER. Can we do something similar to OM? The SCM that is returning the answer will be leader (since read calls go only to Leader), and the other nodes will be followers.

Reply:
Also, can we use org.apache.ratis.proto.RaftProtos.RaftPeerRole#LEADER/FOLLOWER instead of hardcoded strings?